<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GestionnaireTLE</class>
 <widget class="QMainWindow" name="GestionnaireTLE">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>642</width>
    <height>294</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>642</width>
    <height>294</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>642</width>
    <height>294</height>
   </size>
  </property>
  <property name="font">
   <font/>
  </property>
  <property name="windowTitle">
   <string>Gestionnaire de téléchargement des TLE</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../../resources.qrc">
    <normaloff>:/resources/icone.ico</normaloff>:/resources/icone.ico</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="lbl_groupeTLE">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font/>
    </property>
    <property name="text">
     <string>Groupe de TLE :</string>
    </property>
   </widget>
   <widget class="QListWidget" name="listeFichiersTLE">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>30</y>
      <width>200</width>
      <height>220</height>
     </rect>
    </property>
    <property name="font">
     <font/>
    </property>
    <property name="contextMenuPolicy">
     <enum>Qt::CustomContextMenu</enum>
    </property>
    <property name="horizontalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_fichiersTLE">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>10</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font/>
    </property>
    <property name="text">
     <string>Fichiers TLE :</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="MajAutoGroupe">
    <property name="geometry">
     <rect>
      <x>427</x>
      <y>30</y>
      <width>201</width>
      <height>17</height>
     </rect>
    </property>
    <property name="font">
     <font/>
    </property>
    <property name="text">
     <string>Mise à jour automatique du groupe</string>
    </property>
   </widget>
   <widget class="QFrame" name="groupe">
    <property name="geometry">
     <rect>
      <x>427</x>
      <y>50</y>
      <width>211</width>
      <height>211</height>
     </rect>
    </property>
    <property name="font">
     <font/>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="lbl_domaine">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>51</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font/>
     </property>
     <property name="text">
      <string>Domaine :</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="domaine">
     <property name="geometry">
      <rect>
       <x>64</x>
       <y>8</y>
       <width>141</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font/>
     </property>
    </widget>
    <widget class="QLineEdit" name="nomGroupe">
     <property name="geometry">
      <rect>
       <x>96</x>
       <y>33</y>
       <width>109</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font/>
     </property>
    </widget>
    <widget class="QLabel" name="lbl_nomGroupe">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>35</y>
       <width>91</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font/>
     </property>
     <property name="text">
      <string>Nom du groupe :</string>
     </property>
    </widget>
    <widget class="QLabel" name="lbl_listeFichiers">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>111</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font/>
     </property>
     <property name="text">
      <string>Liste des fichiers TLE :</string>
     </property>
    </widget>
    <widget class="QPushButton" name="valider">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>180</y>
       <width>91</width>
       <height>23</height>
      </rect>
     </property>
     <property name="font">
      <font/>
     </property>
     <property name="text">
      <string>Valider</string>
     </property>
    </widget>
    <widget class="QPushButton" name="annuler">
     <property name="geometry">
      <rect>
       <x>114</x>
       <y>180</y>
       <width>91</width>
       <height>23</height>
      </rect>
     </property>
     <property name="font">
      <font/>
     </property>
     <property name="text">
      <string>Annuler</string>
     </property>
    </widget>
    <widget class="QPlainTextEdit" name="listeFichiers">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>195</width>
       <height>91</height>
      </rect>
     </property>
     <property name="font">
      <font/>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="fermer">
    <property name="geometry">
     <rect>
      <x>541</x>
      <y>260</y>
      <width>91</width>
      <height>23</height>
     </rect>
    </property>
    <property name="font">
     <font/>
    </property>
    <property name="text">
     <string>Fermer</string>
    </property>
   </widget>
   <widget class="QListWidget" name="listeGroupeTLE">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>200</width>
      <height>220</height>
     </rect>
    </property>
    <property name="font">
     <font/>
    </property>
    <property name="mouseTracking">
     <bool>true</bool>
    </property>
    <property name="contextMenuPolicy">
     <enum>Qt::CustomContextMenu</enum>
    </property>
    <property name="horizontalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
   </widget>
   <widget class="QCheckBox" name="ageMaxTLE">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>262</y>
      <width>181</width>
      <height>17</height>
     </rect>
    </property>
    <property name="font">
     <font/>
    </property>
    <property name="text">
     <string>Age maximal des TLE (en jours) :</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="nbJoursAgeMaxTLE">
    <property name="geometry">
     <rect>
      <x>195</x>
      <y>260</y>
      <width>42</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font/>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>60</number>
    </property>
    <property name="value">
     <number>15</number>
    </property>
   </widget>
   <widget class="QToolButton" name="creationGroupe">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>10</y>
      <width>20</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="mouseTracking">
     <bool>true</bool>
    </property>
    <property name="autoRaise">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QToolButton" name="ajoutFichiersTLE">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>10</y>
      <width>20</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="mouseTracking">
     <bool>true</bool>
    </property>
    <property name="autoRaise">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="barreMenu">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>642</width>
     <height>22</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI</family>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="nativeMenuBar">
    <bool>false</bool>
   </property>
   <widget class="QMenu" name="menuContextuelGroupe">
    <property name="title">
     <string notr="true">MenuContextuelGroupe</string>
    </property>
    <addaction name="actionCreer_un_groupe"/>
    <addaction name="actionSupprimerGroupe"/>
   </widget>
   <widget class="QMenu" name="menuContextuelFichiersTLE">
    <property name="title">
     <string>MenuContextuelFichiersTLE</string>
    </property>
    <addaction name="actionAjouter_des_fichiers"/>
    <addaction name="actionSupprimer"/>
   </widget>
   <addaction name="menuContextuelGroupe"/>
   <addaction name="menuContextuelFichiersTLE"/>
  </widget>
  <action name="actionCreer_un_groupe">
   <property name="text">
    <string>Créer un groupe</string>
   </property>
  </action>
  <action name="actionSupprimerGroupe">
   <property name="text">
    <string>Supprimer</string>
   </property>
  </action>
  <action name="actionAjouter_des_fichiers">
   <property name="text">
    <string>Ajouter des fichiers</string>
   </property>
  </action>
  <action name="actionSupprimer">
   <property name="text">
    <string>Supprimer</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="../../resources.qrc"/>
 </resources>
 <connections/>
</ui>
